etc/loris2.conf

```diff
@@ -65,6 +65,8 @@ src_img_root = '/usr/local/share/images' # r--
 #cache_root='/usr/local/share/images/loris'
 #user='<if needed else remove this line>'
 #pw='<if needed else remove this line>'
+#cert='<SSL client cert for authentication>'
+#key='<SSL client key for authentication>'
```

Comment:
I think the whole thing could use some cleanup, but--unless I'm missing something--I think the PR as-is matches the existing (albeit somewhat confusing) structure, and I'm :ok:.